### PR TITLE
Create data management UI mockups

### PR DESCRIPTION
**Pull request by marcellp**
# Background
#### Link to issue 
https://biomage.atlassian.net/browse/BIOMAGE-611

#### Link to staging deployment URL 
https://ui-data-management-mockup.scp-staging.biomage.net/

#### Links to any Pull Requests related to this
N/A

#### Anything else the reviewers should know about the changes here
Some of the functionality, such as the modal displayed when clicked on an uploaded or failed file, as well as the dropdown for creating the metadata, are not done here as it would involve writing a lot of dead code.

# Changes
### Code changes

# Definition of DONE
Your changes will be ready for merging after each of the steps below have been completed:

### Testing
- [ ] Unit tests written
- [x] Tested locally with Inframock
- [x] Deployed to staging

To set up easy local testing with inframock, follow the instructions here: https://github.com/biomage-ltd/inframock
To deploy to the staging environment, follow the instructions here: https://github.com/biomage-ltd/biomage-utils

### Documentation updates
Is all relevant documentation updated to reflect the proposed changes in this PR?

- [x] Relevant Github READMEs updated
- [x] Relevant wiki pages created/updated

### Approvers
- [x] Approved by a member of the core engineering team
- [x] (UX changes) Approved by @vickymorrison

### Just before merging:
- [ ] After the PR is approved, the `unstage` script in here: https://github.com/biomage-ltd/biomage-utils is executed. This script cleans up your deployment to staging

### Optional
- [x] Photo of a cute animal attached to this PR
image

### CODE REVIEW

File: src/components/data-management/NewProjectModal.jsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  visible: PropTypes.bool,
+  onCreate: PropTypes.func,
+  onCancel: PropTypes.func,
+  firstTimeFlow: PropTypes.bool,
+};
+
+NewProjectModal.defaultProps = {
+  visible: true,
+  onCreate: null,
+  onCancel: null,
```
Comment (ivababukova) [inline src/components/data-management/NewProjectModal.jsx]: if these are func, then shouldn't their default prop types be equal to `() => null` ?

File: src/components/data-management/ProjectDetails.jsx [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+
+  const [data, setData] = useState([]);
+  const [sortedSpeciesData, setSortedSpeciesData] = useState([]);
+
+  useEffect(() => {
+    if (!speciesData) {
+      return;
+    }
+
+    const commonSpecies = ['hsapiens', 'mmusculus', 'drerio', 'ggallus'];
```
Comment (ivababukova) [inline src/components/data-management/ProjectDetails.jsx:33]: does that mean that if we have user with data that is not from any of these species, we can't support them?

Comment (marcellp) [reply src/components/data-management/ProjectDetails.jsx:33]: No, this just ensures that those species will always be at the top of the list of species in that order, as set out in the wireframes.

File: src/components/data-management/NewProjectModal.jsx (hunk 1 above)
Comment (marcellp) [reply src/components/data-management/NewProjectModal.jsx]: Fixed.

File: src/components/data-management/ProjectDetails.jsx (hunk 2 above)
Comment (ivababukova) [reply src/components/data-management/ProjectDetails.jsx:33]: ah, ok

**Review by ivababukova — APPROVED** [review]
lgtm

**Review by vickymorrison — APPROVED** [review]

**Pull request merged**